protobuf/plugin/raftproxy/raftproxy.go

```diff
@@ -49,21 +49,27 @@ func (g *raftProxyGen) genProxyConstructor(s *descriptor.ServiceDescriptorProto)
 		md["redirect"] = append(md["redirect"], addr)
 		return metadata.NewContext(ctx, md), nil
 	}
-	mods := []func(context.Context)(context.Context, error){redirectChecker}
-	mods = append(mods, ctxMod)
+	remoteMods := []func(context.Context)(context.Context, error){redirectChecker}
```

Comment:
`localCtxMod` is optional, but `remoteCtxMod` is not. I think it makes sense for `remoteCtxMod` not to be optional, because if it was omitted, this would make the requests forwarded by the proxy appear to come from this manager, and that could be disastrous if done by mistake.

We could make `localCtxMod` non-optional as well, but then all the proxy instances that are bound to TCP would have to pass an empty dummy function.

What do you think we should do here?